rllib/evaluation/worker_set.py

```diff
@@ -377,7 +415,7 @@ def set_weight(w):
             self.foreach_worker(
                 func=set_weight,
                 healthy_only=False,
-                remote_worker_indices=to_worker_indices,
+                remote_worker_ids=to_worker_indices,
```

Comment:
Should we rename this arg from `to_worker_indices` to `to_worker_ids` and deprecate the old one, just to signal that there is no longer the concept of a remote worker index?